orquesta/statuses.py

```diff
@@ -67,6 +70,7 @@
     DELAYED,
     RUNNING,
     RESUMING,
+    RERUNNING,
```

Comment:
Remove if these statuses are no longer needed.  If that's the case, rebase the PR to not have a commit to add/delete them.